<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolAddWindow</class>
 <widget class="QDialog" name="ToolAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Add Window</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="text">
      <string>double CLICK</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="14" column="0">
    <layout class="QFormLayout" name="formLayout"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>toolname</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>image</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>produser_name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Inner_diametr</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>outer_diametr</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>legth</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>description</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>INPUT</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>exist produser</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
